pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -149,11 +140,12 @@ public RpcApi convert(final String name) throws RpcApisConversionException {
   // Options parsing is done with CLI library Picocli https://picocli.info/
 
   @Option(
-    names = {"--node-private-key"},
+    names = {"--node-private-key-file"},
     paramLabel = MANDATORY_PATH_FORMAT_HELP,
     description =
         "the path to the node's private key file (default: a file named \"key\" in the Pantheon data folder)"
   )
+  @Nullable
```

Comment:
We aren't typically using the `@Nullable` annotation in Pantheon (at least as far as I can see). It's definitely not the only nullable thing in this class so probably should be marked on just this field in any case.